tests/foreman/longrun/test_inc_updates.py

```diff
@@ -0,0 +1,310 @@
+            act_key,
+            org_name,
+            releasever='6.7'
+        )
+        assert result.return_code == 0
+        result = client.install_katello_agent()
+        client.run('katello-package-upload')
+        return
+
+    @staticmethod
+    def get_applicable_errata(repo):
+        """Retrieves applicable errata for the given repo"""
+        return Errata(repository=repo).search(
+            query={'errata_restrict_applicable': True}
+        )
+
+    def test_api_inc_update_noapply(self):
+        """@Test: Check if api incremental update can be done without
+        actually applying it
+
+        @Feature: Incremental Update
+
+        @Setup:  The prerequisites are already covered in the setUpClass() but
+        for easy debug, get the content view id, Repository id and Lifecycle
+        environment id using hammer and plug these statements on the top of the
+        test. For example::
+
+            self.rhel_6_partial_cv = ContentView(id=38).read()
+            self.rhel_66_repo = Repository(id=164).read()
+            self.qe_lce = LifecycleEnvironment(id=46).read()
+
+        @Assert: Incremental update completed with no errors and Content view
+        has a newer version
+        """
+
+        # Get the content view versions and find the recent one.  API always
+        # returns the versions in ascending order so it is safe to assume the
+        # last one in the list is the recent
+        cv_versions = self.rhel_6_partial_cv.version
+        cvv_id = cv_versions[len(cv_versions) - 1].id
+
+        # Get the applicable errata
+        errata_list = self.get_applicable_errata(self.rhel_66_repo)
+
+        # Apply incremental update using the first applicable errata
+        ContentViewVersion().incremental_update(data={
+            'content_view_version_environments': [{
+                'content_view_version_id': cvv_id,
+                'environment_ids': [self.qe_lce.id]
+            }],
+            'add_content': {
+                'errata_ids': [errata_list[0].id]
+            }
+        })
+
+        # Re-read the content view to get the latest versions
+        self.rhel_6_partial_cv = self.rhel_6_partial_cv.read()
+        self.assertGreater(
+            len(self.rhel_6_partial_cv.version),
+            len(cv_versions)
+        )
+
+    @skip_if_bug_open('bugzilla', 1259057)
+    def test_cli_inc_update_noapply(self):
+        """@Test: Check if cli incremental update can be done without
+        actually applying it
+
+        @Feature: Incremental Update
+
+        @Assert: Incremental update completed with no errors and Content view
+        has a newer version
+
+        """
+        # Get the content view versions and find the recent one.  API always
+        # returns the versions in ascending order so it is safe to assume the
+        # last one in the list is the recent
+
+        cv_versions = self.rhel_6_partial_cv.version
+        cvv_id = cv_versions[len(cv_versions) - 1].id
```

Comment:
Same comment as above. Can we drop `cvv_id` and reference `cv_versions[-1].id` below?